src/utils/autoconnect.ts

```diff
@@ -102,14 +102,11 @@ export const getLastProvider = async (): Promise<Provider | null> => {
 export const setupAutoconnect = async (walletApi: WalletApi): Promise<boolean> => {
   window.addEventListener('beforeunload', () => {
     const { name } = walletApi.getProviderInfo()
-    console.log('LAST PROVIDER NAME', name)
     localStorage.setItem(STORAGE_KEY_LAST_PROVIDER, name)
   })
 
   const provider = await getLastProvider()
 
-  console.log('Connecting to last provider if any', provider)
```

Comment:
Can we make these develop only
```js
process.env.NODE_ENV === 'development' && console.log
```
Sometimes it's useful to see autoconnect working

Reply:
Let me know if this helps: 68200849806ba877b4c2047be2f0eda2a4e702ca